### PR TITLE
Add new zemu tests for set_key function

### LINKED ISSUE #75: session key truncated on ledger screen
Using session.setKeys extrinsic, I only see 5 screens of key on Nano X and 8 screen of keys on Nano S. It's not enough to verify the entire key, especially the last characters. But signature is correct, it's just a display issue.

### PR DESCRIPTION
**Pull request by emmanuelm41**
Closes #75

### CODE REVIEW

**Review by jleni — CHANGES_REQUESTED** [review]
We cannot merge to master unless we first update the version  and recheck the snapshots too as these will affect a few screens in the main menu.

**Comment by jleni** [comment]
I know we didn't update much in the actual code.. but it is better to avoid any chances we confuse releases or change release dates.

We need to update here
https://github.com/Zondax/ledger-kusama/blob/master/app/Makefile.version

the patch version would be enough

**Comment by jleni** [comment]
@emmanuelm41 can you have a look at this? it is not passing CI

**Comment by emmanuelm41** [comment]
> @emmanuelm41 can you have a look at this? it is not passing CI

@jleni, we need to update the test cases with new addresses. Now they are showing extra 0 in the new space added to the parsing process.

**Comment by emmanuelm41** [comment]
I think we are ready here. We need to disable CircleCI too.

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,154 +1,5 @@
 version: 2
```
Comment (jleni) [inline .circleci/config.yml:1]: Any reason why this is still here?
if we deactivate circle.. this will be useless anywauy

File: README.md [hunk 2]
```diff
@@ -25,102 +25,104 @@ Please:
 - **Do not use in production**
 - **Do not use a Ledger device with funds for development purposes.**
 - **Have a separate and marked device that is used ONLY for development and testing**
-
-# Kusama  5.9040.x
+# Kusama  5.9070.x
```
Comment (jleni) [inline README.md:28]:
```suggestion

# Kusama  5.9070.x
```

**Review by jleni — CHANGES_REQUESTED** [review]
if we remove circle. The badge in the README.md needs to be updated/removed

File: .circleci/config.yml (hunk 1 above)
Comment (emmanuelm41) [reply .circleci/config.yml:1]: I removed it on another PR... if we merge this one, and then the other one, we will remove circleci completely.

File: README.md (hunk 2 above)
Comment (emmanuelm41) [reply README.md:29]: Done

**Pull request merged**